Testnet/kichain-t-4/README.md

```diff
@@ -83,45 +84,48 @@ cd $NODE_ROOT
 
 Initiate the needed configuration files using the `unsafe-reset-all` command:
 ```bash
-kid unsafe-reset-all --home ./kid
```

Comment:
It’s some kind of convention in the community to actually keep the `--home` parameter for the unsafe-reset-all command, because this command is _destructive_, and explicit is always better. 
Some chains (like juno) even make the flag mandatory for this command.

I suggest to keep the flag even if it is the default value.